puree/src/main/java/com/cookpad/puree/outputs/PureeBufferedOutput.java

```diff
@@ -26,6 +28,12 @@ public PureeBufferedOutput() {
     @Override
     public void initialize(PureeLogger logger) {
         super.initialize(logger);
+
+        if(storage.count() > MAX_LOG_CAPACITY) {
```

Comment:
I don't like this. We are adding OutSystems logic into the Puree Framework.
This logic must be added to our Logger plugin. We must create a class on our plugin that extends this class and add this logic into it.
Let's try to keep the Puree Framework clean as possible.